manual/development-process/applying-or-destroying-a-bastion-host.md

```diff
@@ -0,0 +1,21 @@
+# Applying or destroying a Bastion host
+
+Using a bastion to connect to the database should be a last resort. For business as usual tasks, we should have API endpoints which provide the required data. 
+## Applying a Bastion host
+
+To bring up a Bastion host you must:
+
+* Go to the Bastion deploy job on [Jenkins](https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Bastion%20Deploy/build)
+* From the "Stage" dropdown, choose the environment that you want to apply the bastion host to
+* From the "Command" dropdown, select "Apply"
+* **Optionally**, if you want to create an ssh tunnel you can add an ssh key. [Instructions to setup the tunnel](./access-an-environment-db.md#5-setting-up-an-ssh-tunnel)
```

Comment:
Where are the instructions for adding the key? The instructions this link to just tell you how to create the tunnel, unless I've missed something?

Reply:
I think I was linking to that to show why you'd want to add the key but maybe that doesn't make sense. I've changed it to say that you need to add a public key to the Jenkins job parameter and added a link for creating an ssh key pair.